addons/mail/static/src/components/channel_preview_view/channel_preview_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.ChannelPreviewView" owl="1">
+        <t t-if="channelPreviewView">
+            <!--
+                The preview template is used by the discuss in mobile, and by the systray
+                menu in order to show preview of threads.
+            -->
+            <div class="o_NotificationListItem o_ChannelPreviewView d-flex flex-shrink-0 align-items-center p-1 cursor-pointer" t-att-class="{ 'o-muted': channelPreviewView.thread.localMessageUnreadCounter === 0 }" t-attf-class="{{ className }}" t-on-click="channelPreviewView.onClick" t-att-data-channel-local-id="channelPreviewView.channel.localId" t-ref="root">
```

Comment:
Git considers this a new file because you did not properly split changes of content of file in a comment and then another commit just for rename of file.

This is a problem with forward-port if you don't do this.
Also the diff gives the wrong impression there are more changes and actually are. It gives less confidence in review that it's the same file with few changes.

Reply:
you are right but it would require not to squash these commits at merge. Sounds ok to you?

Reply:
yes, what I do in my grouped PRs, is to make 2 commits: one for the `[IMP]` and another solely for the `[MOV]`.
That way everything is tracked properly by git.